web/public/index.html

```diff
@@ -49,5 +49,13 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <link rel="stylesheet/less" type="text/css" href="%PUBLIC_URL%/color.less" />
```

Comment:
Why?

Reply:
`color.less` it's a runtime less file. When switch theme modify the variable in it.